src/views/product-tutorials-view/components/product-view-content/components/branded-callout/branded-callout.module.css

```diff
@@ -1,3 +1,7 @@
+.heading {
+  composes: g-offset-scroll-margin-top from global;
```

Comment:
`BrandedCallout` is implemented #375, however there's a URL formatting issue that we need to sort out there first. The intent here is to get the heading working, and leave the chroming work for this component to #375.